### PR TITLE
updated migrations

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/UserService.Data/UserRepository.cs [hunk 1]
```diff
@@ -195,6 +204,10 @@ public bool SwitchActiveStatus(Guid userId, bool status)
             dbUser.IsActive = status;
 
             _provider.Users.Update(dbUser);
+            dbUser.ModifiedBy = _httpContextAccessor.HttpContext.Items.ContainsKey(ConstStrings.UserId) ?
```
Comment (BelehovEgor) [inline src/UserService.Data/UserRepository.cs:207]: strange. how i remember it create user, so userId have to be in context, also in next line you use ?. before HttpContext, but here not, so if httpContext really null you will have HullRefExc before checking

File: src/UserService.Mappers/Db/DbUserCredentialsMapper.cs [hunk 2]
```diff
@@ -24,7 +24,8 @@ public class DbUserCredentialsMapper : IDbUserCredentialsMapper
                 Salt = salt,
                 UserId = request.UserId,
                 PasswordHash = passwordHash,
-                IsActive = true
+                IsActive = true,
+                CreatedAtUtc = DateTime.UtcNow
```
Comment (BelehovEgor) [inline src/UserService.Mappers/Db/DbUserCredentialsMapper.cs:28]: did you forget about CreatedBy or in this models we haven't this property?

File: src/UserService.Mappers/Db/DbUserMapper.cs [hunk 3]
```diff
@@ -68,8 +78,9 @@ public DbUser Map(ICreateAdminRequest request)
                 AvatarFileId = null,
                 IsActive = true,
                 IsAdmin = true,
-                CreatedAt = DateTime.UtcNow,
                 Rate = 1,
+                CreatedBy = userId,
```
Comment (BelehovEgor) [inline src/UserService.Mappers/Db/DbUserMapper.cs:82]: it is not correct, it is id of new user, but user cannot create himself

**Review by BelehovEgor — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

File: src/UserService.Mappers/Db/DbUserCredentialsMapper.cs (hunk 2 above)
Comment (MariyaSin) [reply src/UserService.Mappers/Db/DbUserCredentialsMapper.cs:28]: in this models we haven't this property

File: src/UserService.Data/UserRepository.cs (hunk 1 above)
Comment (MariyaSin) [reply src/UserService.Data/UserRepository.cs:207]: you are right, without "?"

**Review by djino97 — APPROVED** [review]

**Pull request merged**